data/json/vehicles/trucks.json

```
@@ -900,5 +900,138 @@
       { "x": -4, "y": 1, "chance": 10, "//": { "item": "cat_food", "container-item": "can_food" } },
       { "x": -4, "y": 0, "chance": 3, "items": [ "beartrap" ] }
     ]
+  },
+  {
+    "id": "underlift_tow_truck",
+    "type": "vehicle",
+    "name": "Underlift Tow Truck",
+    "blueprint": [
+      [ "     o " ],
+      [ "OHj-+-O" ],
+      [ "OHH'#'|" ],
+      [ "{--'o'>" ],
+      [ "OHH'#'|" ],
+      [ "OHj-+-O" ],
+      [ "     o " ]
+    ],
+    "parts": [
+      { "x": 0, "y": 1, "part": "frame_vertical" },
```

Can you please use modern array-syntax,  e.g.:

```suggestion
      { "x": 0, "y": 1, "parts": [ "frame_vertical", "box", "roof" ] },
```

instead of

```JSON
      { "x": 0, "y": 1, "part": "frame_vertical" },
      { "x": 0, "y": 1, "part": "box" },
      { "x": 0, "y": 1, "part": "roof" },
```
?

---

OK, working on it.

---

If I had the tanks with gasoline inside the array the game would throw an error so I left them as they were.

---

Here is an example:

```{ "x": 1, "y": 0, "parts": [ "frame_handle", { "part": "tank_small", "fuel": "gasoline" } ] },```

---

Fixed the tank lines.